docs/source/usingmame/defaultkeys.rst

```diff
@@ -1,106 +1,558 @@
+
+    When the on-screen display is visible, you can use the following
+    keys to control it:
+
+    * **Up** - select previous parameter to modify.
+    * **Down** - select next parameter to modify.
+    * **Left** - decrease the value of the selected parameter.
+    * **Right** - increase the value of the selected parameter.
+    * **Enter** - reset parameter value to its default.
+    * **Control+Left** - decrease the value by 10x.
+    * **Shift+Left** - decrease the value by 0.1x.
+    * **Alt+Left** - decrease the value by the smallest amount.
+    * **Control+Right** - increase the value by 10x.
+    * **Shift+Right** - increase the value by 0.1x.
+    * **Alt+Right** - increase the value by the smallest amount.
+    * **End** - temporarily hide the On Screen Display.
+    * **Home** - bring the On Screen Display back after hiding it.
+**Up Arrow**
+    Highlight previous UI menu option.
+**Down Arrow**
+    Highlight next UI menu option.
+**Left Arrow**
+    Change current UI option setting when an arrow is present on it.
+**Right Arrow**
+    Change current UI option setting when an arrow is present on it.
+**Home**
+    Highlight first UI menu option.
+**End**
+    Select last UI menu option.
+**Left Shift+Tab**
+    Select previous UI panel.
+**Enter**/**Joystick 1 Button 1**
+    Select currently highlighted UI menu option.
+**Space**
+    Show comment on currently highlighted UI menu option.
+**Delete**
+    Clear/reset to default when highlighting an entry on the input
+    configuration, cheat options, and plugin options pages.
+**[**
+    UI Previous Group
+**]**
+    UI Next Group
+**P**
+    Pauses the emulated machine.
+**Shift+P**
+    While paused, advances to next frame. If rewind is enabled, a new rewind
+    save state is also captured.
+**Shift+~**
+    While paused, loads the most recent rewind save state.
+**-** *(not numeric keypad)*
+    Volume Down for machines that have a volume control.
+**=** *(not numeric keypad)*
+    Volume Up for machines that have a volume control.
+**F1**
+    Power the machine on for machines that have specific power button behavior.
+**F2**
+    Power the machine off for machines that have specific power button behavior.
+
+    Service Mode button for machines that support that.
+**F3**
+    Soft resets the machine.
+**Shift+F3**
+    Performs a "hard reset", which tears everything down and re-creates it from
+    scratch. This is a more thorough and complete reset than the reset you get
+    from hitting F3.
+**LCtrl+F3**
+    Toggle uneven stretch. (*SDL ONLY*)
+**F4**
+    Shows the game palette, decoded graphics tiles/characters and any tilemaps.
+
+    Use the Enter key to switch between the three modes (palette, graphics, and
+    tilemaps).
+
+    Press F4 again to turn off the display.
+    The key controls in each mode vary slightly:
+
+    Palette/colortable mode:
+
+    * **[** **]** - switch between palette and colortable modes.
```

Comment:
Should be “switch between palette devices” or something – I think it’s one per `device_palette_interface` in the system.

Reply:
This isn't a tool I'm familiar with, so I'll take your word on it. Adjusted.